torch/sparse/semi_structured.py

```diff
@@ -404,6 +410,24 @@ def from_dense(
             requires_grad=original_tensor.requires_grad,
         )
 
+    @classmethod
+    def prune_dense_static_sort(cls, original_tensor : torch.Tensor, algorithm="") -> "SparseSemiStructuredTensor":
```

Comment:
Would be it useful to add docs that write out the decomposition?

Reply:
Yeah I can update with the equivalent pytorch ops of what this code is doing and write some docstring.